io.openrct2.OpenRCT2.yaml

```diff
@@ -32,8 +32,7 @@ modules:
     sources:
       - type: git
         url: https://github.com/dscharrer/innoextract.git
-        tag: '1.9'
-        commit: 81fd9b95b76ee5115ce116bfdbc3bc6b887b809d
+        commit: 264c2fe6b84f90f6290c670e5f676660ec7b2387
```

Comment:
We must use a tag in the manifest. Using the 1.9 tag should be fine with innoextract 1.84 as it was previously building.

Reply:
I see